index.html

```diff
@@ -17,6 +17,104 @@
     Header markup here
   </header>
   <main class="dcf-wrapper" id="dcf-main" tabindex="-1" role="main">
+    <!-- Slideshow WIP -->
+    <style>
+      .dcf-slideshow ul {
```

Comment:
Should we put these into dcf scss now or do we plan to do it in phases?  It might be a good learning opportunity for Peter to do it now knowing we will likely tweak it later.

Reply:
@skoolbus39 ^^^ ?

Reply:
@skoolbus39 - these non-theme CSS items are the only things that I would change. They probably should be part of DCF SCSS not here.  If you want to do that in another task that's fine since not sure if Peter is available until Fall. I've already tentatively approved the DCF PR for slideshow depending on this issue.